readthedocs/templates/builds/build_detail.html

```diff
@@ -91,7 +91,11 @@
         {% endif %}
       </span>
       <span class="build-commit" data-bind="visible: commit">
+        {% if build.get_commit_url %}
           (<a data-bind="attr: {href: commit_url}"><span data-bind="text: commit">{{ build.commit }}</span></a>)
+        {% else %}
+            (<span data-bind="text: commit">{{ build.commit }}</span>)
```

Comment:
Does this break the JS code that tries to update `commit_url` if it doesn't exist? I don't know how knockout.js handles this, but we might also need to check for it on that side.

Reply:
Okay I'll check this :)

Reply:
I have checked. there is no error on the browsers console window. and `commit_url` does always exist. but set to `None` when conditions are not met.